app.py

```
@@ -3,6 +3,10 @@
 import numpy as np
 import io
 import os
+import base64
+import openai
```

Ruff (0.8.2)

`openai` imported but unused

Remove unused import: `openai`

(F401)